### PR TITLE
add header, images and styling

### PR DESCRIPTION
**Pull request by codegracie**
Please check my pull request. Thanks.

### CODE REVIEW

**Comment by kian1213** [comment]
Ok

File: app/views/home/signup.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class ="page-wrapper">
+  <div class="center jumbotron">
+    <h1><%= t ".signup_welcome".html_safe %></h1>
+    <h2><%= t ".signup_description".html_safe %>  </h2>
+    <%= link_to "Free Language Learning - Sign up now!", home_path, class: "btn btn-lg btn-primary" %>
```
Comment (MuhammadTamzid) [inline app/views/home/signup.html.erb]: i18n

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -16,27 +16,36 @@
...
-          <h3><%= t ".activities".html_safe %></h3>
-          <ul>
-            <% 5.times do %>
-              <li>
-                <%= link_to image_tag "activities.jpg", size: "50x50" %>
-                <%= t ".sample_activities".html_safe %>
-              </li>
-            <% end %>
-          </ul>
+    <div class="row">
```
Comment (MuhammadTamzid) [inline app/views/home/index.html.erb]: indent

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -16,27 +16,36 @@
...
+      <ul><% 5.times do %><li><%= link_to image_tag "activities.jpg", size: "50x50" %>
+          <%= t ".sample_activities".html_safe %></li>
+          <% end %>
+      </ul>
+    </div>
+    </div>
   </div>
 </div>
 
 <!--For non login user for future updates
```
Comment (MuhammadTamzid) [inline app/views/home/index.html.erb]: remove

File: app/views/home/signup.html.erb (hunk 1 above)
Comment (codegracie) [inline app/views/home/signup.html.erb]: done - included in en.yml

File: app/views/home/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<div class ="page-wrapper">
+  <div class="center jumbotron">
+    <h1><%= t ".about_desc".html_safe %></h1>
```
Comment (dieunb) [inline app/views/home/about.html.erb]: no need `html_safe`

File: app/views/home/about.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+    <h2><%= t ".about_text".html_safe %>  </h2>
```
Comment (dieunb) [inline app/views/home/about.html.erb]: no need `html_safe`

File: app/views/home/index.html.erb [hunk 6]
```diff
@@ -16,27 +16,29 @@
...
-                <%= t ".sample_activities".html_safe %>
-              </li>
-            <% end %>
-          </ul>
+    <div class="row">
+      <div class="col-sm-6 col-md-4"> 
+        <div class="thumbnail"> 
+          <%= link_to image_tag("logo.png", alt: "Words logo", size:"400x400" ) %>
+          <div class="caption">
+            <p><%= link_to t(".button_words").html_safe, words_path, class: "btn btn-primary btn-lg" %></p>
```
Comment (dieunb) [inline app/views/home/index.html.erb]: no need `html_safe`

File: app/views/home/signup.html.erb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<div class ="page-wrapper">
+  <div class="center jumbotron">
+    <h1><%= t ".signup_welcome".html_safe %></h1>
```
Comment (dieunb) [inline app/views/home/signup.html.erb]: no need `html_safe`

File: app/views/home/signup.html.erb [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+    <h2><%= t ".signup_description".html_safe %>  </h2>
```
Comment (dieunb) [inline app/views/home/signup.html.erb]: no need `html_safe`

File: app/views/home/signup.html.erb [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+    <%= link_to t(".signup_button").html_safe, home_path, class: "btn btn-lg btn-primary" %>
```
Comment (dieunb) [inline app/views/home/signup.html.erb]: no need `html_safe`

File: app/views/layouts/_footer.html.erb [hunk 10]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+   <%= t ".small_title".html_safe %>
```
Comment (dieunb) [inline app/views/layouts/_footer.html.erb]: no need `html_safe`

File: app/views/layouts/_footer.html.erb [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+  </small>
+  <nav>
+    <ul>
+      <small>
+        <%= link_to t(".button_about").html_safe, about_path %>
```
Comment (dieunb) [inline app/views/layouts/_footer.html.erb]: no need `html_safe`

File: app/views/layouts/_header.html.erb [hunk 12]
```diff
@@ -0,0 +1,14 @@
+ <!-- Navbar -->
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class ="container">
+  <nav>
+  <%= link_to image_tag("framgia_logo.png", alt: "Framgia logo", size:"50x50" ),  'http://www.fragia.ph' %> 
+    <ul class ="nav bar navbar-nav pull-right">
+      <li><%= link_to "Home",  home_path %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+      <li><%= link_to "Profile", '#' %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+      <li><%= link_to "Categories", categories_path %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+      <li><%= link_to "Results", '#' %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+      <li><%= link_to "Log in", '#' %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 17]
```diff
@@ -0,0 +1,14 @@
+ <!-- Navbar -->
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class ="container">
+  <nav>
+  <%= link_to image_tag("framgia_logo.png", alt: "Framgia logo", size:"50x50" ),  'http://www.fragia.ph' %>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 18]
```diff
@@ -1,14 +1,17 @@
...
-  <%= javascript_include_tag "application", "data-turbolinks-track": true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-  <div class="container upper-margin">
-    <%= yield %>
-  </div>
-</body>
+  <head>
+    <title><%= t ".title".html_safe %></title>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: no need `html_safe`

File: app/views/layouts/_header.html.erb [hunk 19]
```diff
@@ -0,0 +1,14 @@
+ <!-- Navbar -->
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class ="container">
+  <nav>
+  <%= link_to image_tag("framgia_logo.png", alt: "Framgia logo", size:"50x50" ),  "http://www.fragia.ph" %> 
+    <ul class ="nav bar navbar-nav pull-right">
+     <li><%= link_to t(".tablink_home"), home_path %></li>
+     <li><%= link_to t(".tablink_profile"), '#' %></li>
```
Comment (MuhammadTamzid) [inline app/views/layouts/_header.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 20]
```diff
@@ -1,14 +1,19 @@
...
-<body>
-  <div class="container upper-margin">
-    <%= yield %>
-  </div>
-</body>
+  <head>
+    <title>
+      <%= t ".title" %>
+    </title>
+      <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track": true %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/application.html.erb]: indent

File: app/views/layouts/application.html.erb [hunk 21]
```diff
@@ -1,14 +1,19 @@
...
+      <%= javascript_include_tag "application", "data-turbolinks-track": true %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/application.html.erb]: same above

File: app/views/layouts/_header.html.erb [hunk 22]
```diff
@@ -0,0 +1,14 @@
+
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class ="container">
+  <nav>
+    <%= link_to image_tag("framgia_logo.png", alt: "Framgia logo", size:"50x50" ),  "http://www.fragia.ph" %>
+    <ul class ="nav bar navbar-nav pull-right">
+     <li><%= link_to t(".tablink_home"), home_path %></li>
+     <li><%= link_to t(".tablink_profile"), '#' %></li>
```
Comment (MuhammadTamzid) [inline app/views/layouts/_header.html.erb]: double quate

File: app/views/layouts/_header.html.erb [hunk 23]
```diff
@@ -0,0 +1,14 @@
...
+     <li><%= link_to t(".tablink_categories"), categories_path %></li>
+     <li><%= link_to t(".tablink_results"), '#' %></li>
```
Comment (MuhammadTamzid) [inline app/views/layouts/_header.html.erb]: same above

File: config/locales/en.yml [hunk 24]
```diff
@@ -21,15 +21,30 @@
...
+      small_title: Training Project by Grace Sojor and Christian Angel Galamay from Framgia Philippines
+      button_about: About this Project
     admin:
       title: Fels105
       toggle: Toggle Menu
-  partials_admin:
+    partials_admin:
     header:
       title: Admin Page
+<<<<<<< HEAD
```
Comment (dieunb) [inline config/locales/en.yml]: Conflicted code

**Pull request closed without merging**